devenv.bat

```diff
@@ -8,6 +8,7 @@ rem Set default values
 set profiler_platform=x64
 set profiler_configuration=Debug
 set start_visual_studio=true
+set start_rider=false
```

Comment:
Sacrilege! I didn't even know this file existed tbh 🤣

Reply:
I won't complain, it's because @anna-git is team Rider now.